src/components/KeyValueList/index.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Hm, that's an interesting idea. @AlanGreene what do you think about a "How to Use" comment?
I think the storybook sort of fills this role... which reminds me, I will add a storybook entry for the KeyValueList.

Reply:
If this is a purely presentational UI component it should be in packages/components so it can be published and re-used via the @tektoncd/dashboard-components library with our other components.

Yes a story would be the best way to document this, and could provide a few different examples of use if there are interesting variations.

Reply:
Ok, the directory structure has me all confused now 😅 it is a purely presentational UI component, so I will move it to `packages/components`

Reply:
any components remaining in `src/components` should be updated to ensure they're pure and moved to `packages/components` so they can be used by others, looks like we're down to a few SCSS files, the Secrets components, TextInput and TooltipDropdown. I'll create an issue to track the remaining work